contracts/voting-verifier/src/execute.rs

```diff
@@ -162,6 +165,62 @@ pub fn verify_messages(
     ))
 }
 
+/// Returns indices of items in the poll that have achieved quorum, along with their respective results
```

Comment:
On line 175, we get the results of the poll. This returns `Option<Vote>` for each item in the poll, where the value is Some for items that achieved quorum, and None otherwise.